databases_shiny_apps/connect-access-db.R

```diff
@@ -0,0 +1,74 @@
+library(tidyverse) #This package helps import spreadsheet-formatted data, primarily from Excel
+# library(data.table) #Helps with data management
+# library(dbplyr) #This package will help interface with databases by translating R code into database-specific variants
+# library(RODBC)
+library(odbc)
+library(DBI)
+# library(sqldf) # This package might help with more sophisticated SQL queries
+
+
+### with RODBC
+# connection_access <- odbcConnectAccess() ## only available for 32-bit version of Windows.
+# connection <- odbcConnect("<DSN>")
+# query <- "<SQL Query>"
+# data <- sqlQuery(connection, query, )
+# str(data)
+# odbcCloseAll() # At the end of an R session please remember to close the connection. 
+
+#ODBC -  Open Database Connectivity  a standard application programming interface (API) for accessing database management systems
+#DSN - File data sources store the connection information in a text file  not the Windos registry 
+
```

Comment:
This is covered in the docs, so having comments like this here just makes the code more verbose and harder to read.